internal/tools/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/open-telemetry/opentelemetry-specification/internal/tools
 
-go 1.22
```

Comment:
[for reviewer] not related to this PR, but I had to change this otherwise I got `toolchain not available` error when trying to generate the changelog entry template.

Reply:
/cc @joaopgrassi who has the most context on this